content/courses/program-security/signer-auth.md

```diff
@@ -1,61 +1,62 @@
 ---
 title: Signer Authorization
 objectives:
-  - Explain the security risks associated with not performing appropriate signer
-    checks
-  - Implement signer checks using long-form Rust
+  - Explain the security risks of not performing appropriate signer checks.
+  - Implement signer checks using native Rust
   - Implement signer checks using Anchor’s `Signer` type
   - Implement signer checks using Anchor’s `#[account(signer)]` constraint
 description:
-  "Ensure instructions are only ran by authorized accounts by implmementing
-  Signer checks."
+  "Ensure instructions are only executed by authorized accounts by implementing
+  signer checks."
 ---
 
 ## Summary
 
-- Use **Signer Checks** to verify that specific accounts have signed a
-  transaction. Without appropriate signer checks, accounts may be able to
-  execute instructions they shouldn’t be authorized to perform.
-- To implement a signer check in Rust, simply check that an account’s
-  `is_signer` property is `true`
+- **Signer Checks** are essential to verify that specific accounts have signed a
+  transaction. Without proper signer checks, unauthorized accounts may execute
+  instructions they shouldn't be allowed to perform.
+- In Rust, implement a signer check by verifying that an account's `is_signer`
```

Comment:
```suggestion
- In native Rust, implement a signer check by verifying that an account's `is_signer`
```

Reply:
We should probably start with Anchor, since that's the most common path.

Reply:
Updated content to start with Anchor.